cli/Cargo.toml

```diff
@@ -56,14 +56,14 @@ default = [
 ]
 
 stable = [
-    "default"
+    "default",
+    "circuit-template",
```

Comment:
This should probably be 'default' because we want it to always be available in the CLI.